drivers/gpio/gpio_ite_it8xxx2.c

```diff
@@ -233,8 +349,15 @@ static int gpio_ite_configure(const struct device *dev,
 	volatile uint8_t *reg_gpdr = (uint8_t *)gpio_config->reg_gpdr;
 	volatile uint8_t *reg_gpcr = (uint8_t *)(gpio_config->reg_gpcr + pin);
 	volatile uint8_t *reg_gpotr = (uint8_t *)gpio_config->reg_gpotr;
+	volatile uint8_t *reg_1p8v = &IT8XXX2_GPIO_GCRX(
+				gpio_1p8v[gpio_config->index][pin].offset);
+	volatile uint8_t mask_1p8v =
+				gpio_1p8v[gpio_config->index][pin].mask_1p8v;
```

Comment:
declare the variables reg_1p8v/mask_1p8v here without the assignment

Reply:
Done.